### ISSUE TITLE
Batch norm on shortcut in Resnet

### ISSUE THREAD

**Issue by gussmith23** [kuangliu/pytorch-cifar#100]
Is it normal to have a batch norm on the shortcut path in Resnets? The paper states that they apply batch normalization after each convolution, and there is technically a 1x1 convolution on each shortcut path, so I understand where this interpretation might be coming from.

**Comment by gabrieldernbach** [comment]
I am curious too, did you compare with and without?
